code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1,9 +1,12 @@
-/mob/living/carbon/Initialize()
+/mob/living/carbon/Initialize(mapload)
 	. = ..()
 	create_reagents(1000)
 	assign_bodypart_ownership()
 	update_body_parts() //to update the carbon's new bodyparts appearance
 	GLOB.carbon_list += src
+	if(mapload) //I don't want no gas leaks on my space ruin you hear?
```

Comment:
I'm pretty sure the answer is no, but does preserve the default morgue corpse bodies creating miasma?

Reply:
mmmm not sure, might prevent it. why?

Reply:
Idunno, it feels like if your problem is with space ruins then you might want to keep an eye out for anything that this changes that conflicts with that.

Reply:
Notably gas miners, found in the cat surgeon and syndibase ruins